src/sinks/aws_cloudwatch_logs/mod.rs

```diff
@@ -58,8 +59,7 @@ pub struct CloudwatchLogsSinkConfig {
     pub encoding: Encoding,
     pub create_missing_group: Option<bool>,
     pub create_missing_stream: Option<bool>,
-    pub batch_timeout: Option<u64>,
-    pub batch_size: Option<usize>,
+    pub batch: Option<BatchConfig>,
```

Comment:
Would it be cleaner to do

```rust
#[serde(default)]
pub batch: BatchConfig,
```

for these? With a `#[derive(Default)]` on `BatchConfig` you'd just get `None` on every field, which is one layer of `Option` instead of the current two.

Reply:
Absolutely, thanks for the pointer!